react/src/pages/Pogoda_term.js

```diff
@@ -1,20 +1,59 @@
 import Navbar from '../components/Navbar';
 import Header from '../components/Header';
 import ButtonsPogoda from '../components/ButtonsPogoda';
-function PogodaTermPage(){
-    return (
-        <div className="App">
-          <div id="header"> 
-            <Header />
-          </div>
-          <div id="menu">
-            <Navbar />
-          </div>
-          <div>
-            <ButtonsPogoda />
-          </div>
-        </div>
-      );
+import Search from '../components/Weather/Search';
+import CurrentWeather from '../components/Weather/CurrentWeather';
+import { weatherApi } from '../components/Weather/Api';
+import { useState } from 'react';
+
+function PogodaTermPage() {
+
+  const [currentWeather, setCurrentWeather] = useState(null);
+  const [forecast, setForecast] = useState(null);
+
+
+  const handleOnSearchChange = (searchData) => {
+    const [lat, lon] = searchData.value.split(" ");
+
+    const currentWeatherFetch = fetch(
+      `${weatherApi.base}/weather?lat=${lat}&lon=${lon}&appid=${weatherApi.key}&units=metric`
+    );
+    const forecastFetch = fetch(
+      `${weatherApi.base}/forecast?lat=${lat}&lon=${lon}&appid=${weatherApi.key}&units=metric`
+    );
```

Comment:
`currentWeatherFetch` => `fetchCurrentWeather`
`forecastFetch` => `fetchForecast`

To szczegół, ale to tak jakbyśmy nazwali funkcję sumującą `sumGet` zamiast `getSum`